### DISCUSSION TITLE
Memory card questions

### DISCUSSION THREAD

**Discussion by isaiahfisher** [emukidid/swiss-gc#921] [Q&A]
I have been digging through the docs and the open issues on memory cards and I am a little confused. 

1) Let's say I have games from USA region and Japan reason. If I use a physical memory card with these games and Swiss, does Swiss somehow magically prevent formatting / corruption of the physical memory card or do I need two separate physical cards?

2) In the case of memory card emulation, do I need to manually build the virtual memory cards for the 3 regions myself then enable the setting, or does enabling the setting automatically create the cards?

3) If I am rocking my Swiss SD card in my SD2SP2 adapter and using Picoboot to bypass the onboard, can I use virtual memory card emulation directly on the same SD? Are there major risks with doing so? I am also storing my games on this same SD under a 'games' folder, so I would imagine the virtual cards could go under a 'saves' folder?

4) Can I copy saves back and forth between physical cards and virtual cards?

Thanks in advance for helping me understand these issues. If any of these are not possible but could realistically be worked on, I would be happy to start investigating a potential solution and PR if you could point me in the right direction to begin familiarizing myself.

**Comment by Extrems** [comment]
1. It's prevented.
2. They're automatically created but not formatted, so you'll be prompted to format by the game.
3. It's the only choice.
4. Not easily. You can use boot through IPL to manage them, but copy/move protection is still applicable.

**Reply by rapperskull** [reply]
Maybe a little bit off-topic, but I think it would be useful to have a "Boot to IPL" button or shortcut, in order to boot the console to IPL without having to enable the "Boot through IPL" option, launch a game and remember to hold the button.

**Reply by isaiahfisher** [reply]
Thanks for the answers! So I can just store the save files on the same physical memory card or by prevented you mean Swiss prevents the save from hitting that card?

As for point 4, is there a reasonable path forward for making an interface to manage it? I would be happy to take a shot at it, though it has been awhile since I touched C programming.

**Reply by Extrems** [reply]
It will save as normal.

**Reply by isaiahfisher** [reply]
Sweet thanks! I appreciate the work you guys have put into this. Its crazy awesome. If there is any issue well suited to someone new that I could try to tackle I would love to contribute back.
